translations/maintainer_guide/maintainer_guide_hin.md

```diff
@@ -0,0 +1,61 @@
+# अनुरक्षक की मार्गदर्शिका
+यह मार्गदर्शिका उन लोगों के लिए है जो समुदाय के लिए परियोजना के प्रबंधन और रखरखाव में सहायता के लिए अनुरक्षक के रूप में परियोजना में शामिल होना चाहते हैं। (यह **पहली बार योगदान करने वालों** के लिए मार्गदर्शिका नहीं है)
+## उद्देश्य
```

Comment:
The Translate section is missing. You don't need to add the table, but add the translation section with other properties.